src/domain/catalog/services/menu.service.ts

```diff
@@ -90,45 +64,74 @@ export class MenuService {
     }
   }
 
-  async findAll(): Promise<Menu[]> {
+  async findAll(businessId: string): Promise<Menu[]> {
```

Comment:
Should this be findAllWithCategories since you are populating with categories ?
```suggestion
  async FindAllWithCategories(businessId: string): Promise<Menu[]> {
```